platform-emf/src/main/java/com/softicar/platform/emf/management/EmfManagementDiv.java

```diff
@@ -236,7 +236,7 @@ private boolean isCreationAllowed() {
 			if (scopeEntity != null) {
 				return entityTable.isCreationPossible(scopeEntity) && entityTable.isCreationAllowed(scopeEntity);
 			} else {
-				return CurrentBasicUser.get().isSuperUser();
+				return false;
```

Comment:
is this correct? can admins still create users with this change?

Reply:
didnt you say yesterday to just return false here?
And what does this have to do with users?

Reply:
but yes, I can still create users when I log into the admin user